test/unit/throw.spec.js

```diff
@@ -26,6 +26,24 @@ describe('a test that throws', function () {
     });
   });
 
+  describe('extensible', function () {
+    it('should not crash if throwing non-extensible type', function (done) {
+      var test = new Test('im async and throw string async', function (done2) {
```

Comment:
`done2` is unused.

Reply:
Neither it is in several places in the same file (I just reused an existing test), see :
*  [L63](https://github.com/fargies/mocha/blob/41fe7637a2f833689b47b75bdb2650e6ba85603f/test/unit/throw.spec.js#L63)
*  [L77](https://github.com/fargies/mocha/blob/41fe7637a2f833689b47b75bdb2650e6ba85603f/test/unit/throw.spec.js#L77)
*  [L109](https://github.com/fargies/mocha/blob/41fe7637a2f833689b47b75bdb2650e6ba85603f/test/unit/throw.spec.js#L109)
*  [L123](https://github.com/fargies/mocha/blob/41fe7637a2f833689b47b75bdb2650e6ba85603f/test/unit/throw.spec.js#L123)

The linter doesn't seem to report it for some reason.

Reply:
It's normal to have `done2` argument there to make the test an async test.

I added eslint-ignore comments.

Reply:
Ooops! I got it.
It needed to make the test as async as you said.

Reply:
The comment makes it clearer, so it's better anyway :)